deploy/k8s/manifest.yaml

```diff
@@ -375,7 +375,24 @@ data:
               - action: update_label
                 label: owner_name
                 new_label: deployment
-
+          - include: k8s.container_fs_(reads|writes)_total
+            action: insert
+            match_type: regexp
+            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's CPU usage
+            experimental_match_labels: { "image": "^$", "pod": "(.|\\s)*\\S(.|\\s)*", "namespace": "(.|\\s)*\\S(.|\\s)*" }
+            new_name: k8s.pod.fs.iops
```

Comment:
What is this metric supposed to contain? If I read the code right, we take metrics `k8s.container_fs_reads_total` and `k8s.container_fs_writes_total`, filter them and rename them to `k8s.pod.fs.iops`. So that the new metric will have data points mixed from datapoints of the original metrics. Did you want to sum them together? Or name the new metric differently?

Reply:
Yes, the intention is to combine reads and writes into a single metric. I can see now, that's not what I am achieving here, as reads and writes are essentially just renamed and never summed up. Could you advice how to achieve (reads + writes)? The same intention was for reads and writes bytes total.